ayon_api/entity_hub.py

```diff
@@ -2419,10 +2437,13 @@ def get_path(self, dynamic_value=True):
 
         if self._path is None:
             parent = self.parent
-            path = self.name
             if parent.entity_type == "folder":
                 parent_path = parent.path
-                path = "/".join([parent_path, path])
+                path = "/".join([parent_path, self.name])
+            elif self._entity_hub.path_start_with_slash:
+                path = "/{}".format(self.name)
+            else:
+                path = self.name
```

Comment:
Shouldn't this be something like:
```py
if parent.entity_type == "folder":
    parent_path = parent.path
    path = "/".join([parent_path, self.name])
else:
    path = self.name

if self._entity_hub.path_start_with_slash:
    path = f"/{path}"
```

As in we need to pretend slash to both folders and entities?

Reply:
No, only the root folder must add slash at the beginning. Other are using parent path which already has slash at the beginning.